gui/app/services/settingsService.js

```diff
@@ -320,6 +320,24 @@
                 pushDataToFile("/settings/activeChatUsers/inactiveTimer", inactiveTimer);
             };
 
+            service.getActiveMixplayUserList = function() {
+                let status = getDataFromFile("/settings/activeMixplayUsers/status");
+                return status != null ? status : "On";
+            };
```

Comment:
See the comment I left on the backend version of this method (settings-access.js > getActiveMixplayUserList). I suggest making this save and return a boolean instead of a string that acts like a boolean. Its less confusing and it looks like other parts of your changes are assuming this returns a boolean anyway. 

Also rename to `getActiveMixplayUserListEnabled` for clarity.

Reply:
This particular bit isn’t that important. I mainly just wanted to point out that we don’t need to call parseInt before calling isNaN. Doesn’t change much in this particular example but it can simply code a bit in other situations.